auth/src/commonMain/kotlin/com/liftric/auth/core/Constants.kt

```diff
@@ -0,0 +1,83 @@
+package com.liftric.auth.core
+
+internal object Header {
+    const val Authority = "authority"
+    const val CacheControl = "cache-control"
+    const val AmzUserAgent = "x-amz-user-agent"
+    const val Useragent = "user-agent"
+    const val Accept = "accept"
+    const val SecFetchSite = "sec-fetch-site"
+    const val SecFetchMode = "sec-fetch-mode"
+    const val AcceptLanguage = "accept-language"
+    const val Dnt = "dnt"
+    const val AmzTarget = "x-amz-target"
+    const val AmzJson = "application/x-amz-json-1.1"
+}
+
+internal enum class Authentication(val flow: String) {
+    RefreshTokenAuth("REFRESH_TOKEN_AUTH"),
+    UserPasswordAuth("USER_PASSWORD_AUTH")
+}
+
+/**
+ * @see https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints
+ */
+enum class Region(val code: String) {
```

Comment:
I just thought whether this is actually necessary! Having just the region string could make things easier. Additionally we wouldn't have to alter that part for future regions. Currently I cannot feel any benefit from that @Ingwersaft @gaebel. Important: weak opinion!

Reply:
We could also provide a secondary constructor, but I agree that it is not providing a lot of value.

Reply:
Yeah, just use string directly, AWS will provide a proper error in case of typos :)